cms_form/templates/form.xml

```diff
@@ -235,4 +235,49 @@ License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
 </template>
 
 
+<template id="wizard_form_wrapper" name="CMS wizard form wrapper"
+          inherit_id="cms_form.form_wrapper" primary="True">
+
+  <t t-if="form.form_title" position="before">
+    <t t-if="form.wiz_show_progress_bar">
```

Comment:
Nitpicking : Here we get to `t` elements with `t-if` right after, but one is a selector and the second an addition. Using `xpath`element for first would make this more obvious.

Reply:
I avoid xpaths when possible, I think this is more readable ;P